doc/stdenv/stdenv.xml

```diff
@@ -2111,16 +2111,106 @@ postInstall = ''
       </para>
      </listitem>
     </varlistentry>
+
     <varlistentry>
      <term>
-      cmake
+      <anchor xml:id="cmake"/>
```

Comment:
You can add `xml:id` attribute directly to `term` element.

Reply:
yea, I thought about that after I created the PR

Reply:
do you know a way that i can get the "link" element without using a section?

Reply:
What do you mean by using a section?

You can make any element linkable by adding `xlink:href="#targetid` attribute, or if you want to linkify arbitrary text, wrap it with `<link>` element and add the `xlink:href` argument to that. Or if you want to automatically generate the link title, use `<xref linkend="targetid" />` – that should work for chapter, sections, tables, terms…

Reply:
I meant like https://nixos.org/manual/nixpkgs/stable/#chap-overlays where if you hover over the text, it will give you a "link" to that location

Reply:
xlink:href might work though, as that will at least make the text a link. better than the anchor to nowhere at least

Reply:
Oh, right, we do not have that. Maybe I could try to write a XSL transformation later but not sure if it is worth the effort with impending markdownification. For now, I just use developer tools to get the anchor links. And there are probably browser extensions that can make it easier too,